python/graphscope/learning/graph.py

```diff
@@ -56,9 +56,47 @@ def __init__(self, session, graph, nodes=None, edges=None, gen_labels=None):
             session (:class:`Session`): instance of GraphScope session.
             graph (:class:`graphscope.framework.graph.GraphDAGNode`):
                 Source property graph.
-            nodes (list): The node types that will be used for gnn training.
-            edges (list): The edge types that will be used for gnn training.
-            gen_labels (list): Extra node and edge labels on original graph for gnn training.
+            nodes (list, optional): list of node types that will be used for gnn
+                training, the element of list can be `node_label` or
+                `(node_label, features)`. If the element of list is a tuple and
+                contains selected feature list, it would use the selected
+                feature list for training. if nodes is None, use all type of
+                nodes and features for the gnn training.
+            edges (list, optional): list of edge types that will be used for gnn
+                training. we use `(src_label, edge_label, dst_label)`
+                to specify one edge type. if edges is None, use all type of
+                edges for gnn training.
+            gen_labels (list, optional): Alias node and edge labels and extract
+                train/validation/test dataset from original graph for supervised
+                gnn training. We will explains the details in the examples.
```

Comment:
Better to use the declarative tone: The detail is explained in the examples below.

Reply:
fixed